package.json

```diff
@@ -52,7 +52,7 @@
     "compile"
   ],
   "dependencies": {
-    "@socket.tech/dl-core": "https://github.com/KintoXYZ/socket-DL.git",
+    "@socket.tech/dl-core": "https://github.com/KintoXYZ/socket-DL.git#second-signer",
```

Comment:
Change to be removed once `second-signer` branch is merged or when all kinto utils inside socket-DL are extracted to a separate `kinto-utils` repo.